workflows/proteomics/metaproteomics-gtn/metaproteomics_gtn_wf.ga

```diff
@@ -0,0 +1,724 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "This workflow identifies expressed proteins from a complex bacterial community sample. For this, MS/MS data will be matched to peptide sequences provided through a FASTA file.\n",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0001-6801-2559",
+            "name": "Timothy Griffin"
+        },
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0003-0984-0973",
+            "name": "Pratik Jagtap"
+        },
+        {
+            "class": "Person",
+            "identifier": "https://github.com/jj-umn",
+            "name": "James Johnson"
+        },
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0002-1726-2256",
+            "name": "Clemens Blank"
+        },
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0001-9818-0537",
+            "name": "Subina Mehta"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "release": "0.1",
+    "name": "Metaproteomics workflow",
```

Comment:
That's a *very* generic name. Can it be turned into something a bit more specific to distinguish it from similar wfs we might have at some point?

Reply:
Done